### DISCUSSION TITLE
Non riesco a connettermi ad homeassistant.local dopo reinstallazione

### DISCUSSION THREAD

**Discussion by Andrew186** [indomus/forum#1926] [HUB: Home Assistant]
Salve a tutti sono nuovo e partecipo al forum subito con un bel problema.

Da oggi ho reinstallato da zero home assistant sul mio raspberry, ma dopo aver formattato e flashato la sd non riesco più ad entrare homeassistant.local:8123 o homeassistant:8123 mi viene fuori la scritta "Impossibile raggiungere il sito"

Potete aiutarmi a risolvere il problema?

Grazie a tutti

**Comment by Moxser** [comment]
Ciao 
Hai provato a inserire l'indirizzo IP che hai impostato sul tuo raspberry http://192.168.0.---:8123

**Reply by Andrew186** [reply]
Grazie per la risposta, purtroppo non funzionava neanche con l'IP.
Alla fine ho risolto flashando una vecchia versione di Home Assistant e aggiornandola alla nuova versione, non so come mai ma funziona.

Purtroppo ora ho un nuovo problema, ossia che non installa l'aggiornamento di Home Assistant Core Update in versione 2022.9.7
